package.json

```diff
@@ -3,6 +3,9 @@
   "version": "1.3.5",
   "description": "AST utility module for statically analyzing JSX",
   "main": "lib/index.js",
+  "files": [
+    "lib"
```

Comment:
In order to test the compiled package before publishing in a programmatic way, a project would need to add a test that runs `npm pack`, untars the resulting tarball in a temp dir, cd's into it, and then what? If tests are published, it could run `npm install && npm test`, but if tests are not published, how would the tests be able to run against that packed module?

The only way I can see this working is if all tests run `npm pack`, untar, and import *that* to run tests against, ie, never running tests against the original source.

Can you point me to a single project that tests its published output? I can point to many projects that have screwed up what gets published such that the published package is unknowingly broken :-/